### PR TITLE
Update bootstrap with meta

### PR DESCRIPTION
**Pull request by s-pace**
Waits for twbs/bootstrap#27902

### CODE REVIEW

File: configs/webdriver.json [hunk 1]
```diff
@@ -2,24 +2,20 @@
   "index_name": "webdriver",
   "start_urls": [
     {
-      "url": "http://webdriver.io/guide.html",
+      "url": "http://beta.webdriver.io/docs/gettingstarted.html",
```
Comment (XhmikosR) [inline configs/webdriver.json]: It seems you modified webdriver.json here too @s-pace

Comment (s-pace) [reply configs/webdriver.json]: Yep under control, I will discard these changes

**Comment by XhmikosR** [comment]
hey @s-pace. We'll release v4.3 later or today, 🤞 which will include the 2 meta tags.

**Comment by s-pace** [comment]
Great @XhmikosR 🎉 

Do the pages from version 4.1 and 4.0 will also includes these meta tags?

Ping me when it is ready, we will deploy it asap

**Comment by XhmikosR** [comment]
No, the old versions, none of them, will change, because this requires rebuilding the old docs and it's quite troublesome.

**Comment by s-pace** [comment]
@XhmikosR Thanks for the details, I will do a clearer PR that will only affect the latest version.

**Comment by XhmikosR** [comment]
Thanks! Please CC me there so that i can ping you.

**Pull request closed without merging**